### ISSUE TITLE
Color Contrast ratio of focus indicator on close (X) button is less than required ratio 3:1.

### ISSUE THREAD

**Issue by msftedad** [flutter-stripe/flutter_stripe#1921]
**Describe the bug**
Color Contrast ratio of focus indicator on close (X) button is less than required ratio 3:1.

**To Reproduce**

1. Open URL: https://site-ux5ko.powerappsportals.com/pay/ in Edge browser.
2. A 'Pay' page appears. Fill Name, Amount and currency as USD and click on next button.
3. Navigate to 'Cash app pay' tab using tab key and activate it using Enter key.
4. Navigate to 'Pay now' control using tab key and activate it using Enter key. 'Scan...' popup appears. 
5. Open Accessibility insights for windows and check color contrast ratio of focus indicator on close (x) button.
6. Color contrast ratio of focus indicator on close (x) button is less than required 3:1 ratio or not.

**Expected behavior**
Color Contrast ratio of focus indicator on close (x) button should be greater than or equal to the required ratio 3:1.

Windows 11 24H2 
Os build: 26100.1586
Browser: Edge Version 128.0.2739.54 (Official build) (64-bit)

**Additional context**
<img width="958" alt="Color Contrast ratio of the focus indicator on 'X' button is less than required ratio  (1)" src="https://github.com/user-attachments/assets/aabc232c-b025-4d08-8322-0722ae77af4e">

**Comment by msftedad** [comment]
@remonh87 - Is this issue fixed?

**Comment by remonh87** [comment]
these are external stripe test windows that are outside of this library

**Comment by msftedad** [comment]
@remonh87 - We have 9 issues for Stripe, we connected with Stripe support team, and they suggested to log these issues on https://github.com/flutter-stripe/flutter_stripe/ library. Please confirm where we should log and track these bugs?

**Comment by remonh87** [comment]
When it comes to issues with Flutter sdk you can log them to us. The issues that you are logging are testscreens maintained by stripe so you need to log them there.  However since they are not even real production screens I highly doubt stripe will fix it

**Comment by msftedad** [comment]
@remonh87 - Thank you for the explanation, we will reach out to stripe support for these issues.
